apps/web/components/push-permission-prompt.tsx

```
@@ -0,0 +1,89 @@
+      role="alert"
+      className="fixed bottom-20 left-4 right-4 z-[9998] mx-auto max-w-sm rounded-xl border p-4 shadow-xl md:left-auto md:right-6 md:bottom-6"
+      style={{
+        background: "var(--theme-bg-secondary)",
+        borderColor: "var(--theme-bg-tertiary)",
+      }}
+    >
+      <button
+        onClick={dismiss}
+        className="absolute top-2 right-2 p-1 rounded-md hover:opacity-80"
+        aria-label="Dismiss notification prompt"
+      >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)